src/main/java/com/jfrog/ide/idea/ui/configuration/XrayGlobalConfiguration.java

```diff
@@ -84,6 +84,10 @@ public XrayGlobalConfiguration() {
                 username.setEnabled(true);
                 url.setEnabled(true);
                 password.setEnabled(true);
+                XrayServerConfigImpl oldConfig = GlobalSettings.getInstance().getXrayConfig();
```

Comment:
Let's add a comment describing why we do this.

Reply:
Added 👍